fem/bilininteg.hpp

```diff
@@ -3241,6 +3281,24 @@ class DGDiffusionIntegrator : public BilinearFormIntegrator
                                    const FiniteElement &el2,
                                    FaceElementTransformations &Trans,
                                    DenseMatrix &elmat);
+
+   virtual bool RequiresFaceNormalDerivatives() const { return true; }
```

Comment:
Let's replace `virtual` with `override` everywhere in this class.

Reply:
Good idea, done in 16b3b6854.